mixer/doc/adapter-development-walkthrough.md

```diff
@@ -488,10 +475,12 @@ spec:
 
 # Step 7: Start Mixer and validate the adapter.
 
-Start the mixer pointing it to the sample operator configuration
+Start the mixer pointing it to the testdata operator configuration
 
 ```bash
-pushd $MIXER_REPO && go install ./... && mixs server --configStoreURL=fs://$MIXER_REPO/adapter/mysampleadapter/sampleoperatorconfig
+pushd $ISTIO/istio && make mixs
+// locate mixs binary, should be $GOPATH/out/linux_amd64/release/mixs
```

Comment:
nit: we should clarify that this is the output directory for certain linux architectures. we probably want a disclaimer for mac too.

Reply:
Thanks.. Fixed